src/backend/app.py

```diff
@@ -21,31 +23,59 @@ def update_new_receipts(name):
 
 @app.route('/api/insert_receipts_to_db')
 def insert_receipts_to_db():
-    # process receipts to pandas df
-    item_df = FS.receipts_to_dataframe()
+    FS.setup('alex') 
```

Comment:
The login URL needs to set the profile ID of the session. This directs the filesystem to the Alex directory.

Instead of doing this, write a function that processes this for you and then just returns Alex.